ui/stack.dev.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: lemmy-ui--dev
+spec:
+  selector:
+    matchLabels:
+      app: lemmy-ui--dev
+  template:
+    metadata:
+      labels:
+        app: lemmy-ui--dev
+    spec:
+      containers:
+        - name: lemmy-ui--dev
+          image: registry.gitlab.com/pojntfx/lemmy/ui.dev
+          resources:
+            limits:
+              memory: 1024Mi
+              cpu: 512m
+          ports:
+            - containerPort: 4444
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: lemmy-ui--dev
+spec:
+  type: NodePort
+  selector:
+    app: lemmy-ui--dev
+  ports:
+    - port: 4444
+      nodePort: 30002
```

Comment:
Looks pretty neat, so this is used for development too? interesting.

Reply:
Yes, this is being used for everything, the whole workflow. I'm still implementing the production version though, currently this is dev only (port `4444` gets forwarded to localhost and hot-reloading works which makes this a very viable alternative to "native" development).